lib/internal/fs/promises.js

```diff
@@ -106,8 +106,8 @@ class FileHandle extends JSTransferable {
     return fsync(this);
   }
 
-  read(buffer, offset, length, position) {
-    return read(this, buffer, offset, length, position);
+  read(...args) {
+    return read(this, ...args);
   }
```

Comment:
I would leave this signature alone. It's subtle, and not likely to break anyone, but changing this does not a small impact on the API and it's just not really necessary.

Reply:
Yeah, I agree, it does look more nicer and cleaner the way it originally was. I would rather manually parse parameters. Your thoughts @Himself65, @mscdex?

Reply:
Another implementation is to parse the parameters in this class function.
Whatever, I’m 👎 with the previous handle which uses `filter`